front/src/reducers/user/userReducer.spec.ts

```diff
@@ -1,104 +1,69 @@
-import { expect, test } from 'vitest';
+import { expect, it } from 'vitest';
 import { createStoreWithoutMiddleware } from 'Store';
 import {
   userInitialState,
   loginSuccess,
   loginError,
-  logoutUser,
-  serverError,
-  updateAccount,
+  logoutSuccess,
   UserState,
 } from 'reducers/user';
+import { ApiError } from 'common/api/baseEditoastApi';
 
 const createStore = (initialStateExtra?: UserState) =>
   createStoreWithoutMiddleware({
     user: initialStateExtra,
   });
 
 describe('userReducer', () => {
-  test('should return the initial state', () => {
+  it('should return the initial state', () => {
     const store = createStore(userInitialState);
     const userState = store.getState().user;
     expect(userState).toEqual(userInitialState);
   });
 
-  test('should handle loginSuccess', () => {
+  it('should handle loginSuccess', () => {
     const store = createStore(userInitialState);
-    const action = loginSuccess({ accessToken: 'fake_token', username: 'Test userSlice' });
+    const action = loginSuccess({ username: 'Test userSlice' });
     store.dispatch(action);
     const userState = store.getState().user;
     expect(userState).toEqual({
       isLogged: true,
-      toLogin: false,
-      loginError: false,
-      serverError: false,
+      loginError: undefined,
       username: 'Test userSlice',
-      accessToken: 'fake_token',
       account: {},
     });
   });
 
-  test('should handle loginError', () => {
+  it('should handle loginError', () => {
     const store = createStore(userInitialState);
-    const action = loginError(true);
+    const error = new ApiError(
+      {
+        type: 'error_type',
+        message: 'message',
+        context: {},
+      },
+      502
+    );
+    const action = loginError(error);
     store.dispatch(action);
     const userState = store.getState().user;
     expect(userState).toEqual({
       isLogged: false,
-      toLogin: true,
-      loginError: true,
-      serverError: false,
+      loginError: error,
       username: '',
-      accessToken: undefined,
       account: {},
     });
   });
 
-  test('should handle logoutUser', () => {
+  it('should handle logoutUser', () => {
     const store = createStore({
+      ...userInitialState,
```

Comment:
Can you add a username not empty here ?
```suggestion
      ...userInitialState,
      username: 'test logout'
```